test/serializer.spec.ts

```diff
@@ -219,5 +223,24 @@ describe('Serializer service', () => {
                 child: 'grandchild'
             }, ParentExample).test()).to.eql('GRAND CHILD');
         }));
+
+        it('Should throw an error if no child is found with given discriminator', () => {
+            serializer.register([
```

Comment:
I think we should avoid providing too much fake registers for a test that doesn't use any of them